product_docs/docs/pgd/5/pgdlifecycle.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: PGD lifecycle
+---
+
+## Introduction
+
+As part of any planning process, it is essential to know the supported platforms and planned supported life of the application. This section provides those details for PGD 5.x.
+
+## PGD 5.x Lifecycle
+
+| Event | When |
+| --- | --- |
+| General availability | 21 Feb 2023 |
+| Standard support ends | 18 months after the release of 6.0 |
+
+
+## PGD 5.x Supported CPU architectures and OS
+
+### Linux x86-64 (Amd64)
+
+| Operating System | Versions&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  |
+| ---------------- | -------- |
+| RHEL | 9/8/7 |
+| OL | 9/8/7 |
+| RockyLinux/AlmaLinux | 9/8 |
```

Comment:
Can we lump these together as "Red Hat Enterprise Linux (RHEL) and compatible derivatives (Alma Linux, Oracle Linux, Rocky Linux)"?

Reply:
No, because of the fracture. Rocky and Alma only go back to 8, CentOS is still listed and stops at 7.

This is in sync with the compatibility table on the main site and I'd rather it not go out of sync with its versioning layout.

And also here's hoping we can fill it out with some ARM soon :)